src/lib/styles/PostPreviewCard.styles.ts

```diff
@@ -1,27 +1,51 @@
-import { makeStyles } from '@material-ui/core';
+import { makeStyles, Theme } from '@material-ui/core';
 
 export const useStyles = makeStyles(
-  {
+  (theme: Theme) => ({
     root: {
       position: 'relative',
-      height: 265,
+      width: 280,
+      borderRadius: 0,
     },
     media: {
-      height: 58,
-      width: 46,
-      padding: 15,
-      paddingLeft: 13,
+      height: 91,
+      width: 91,
+      padding: theme.spacing(3),
+      paddingLeft: theme.spacing(3),
+      filter: 'grayscale(100%)',
     },
     header: {
       display: 'flex',
-      margin: '15px 15px 0 15px',
+      margin: theme.spacing(0, 4),
       cursor: 'pointer',
     },
     body: {
-      margin: '15px',
+      margin: theme.spacing(1, 4, 5),
       cursor: 'pointer',
     },
-  },
+    postType: {
+      color: theme.palette.info.light,
+    },
+    authorTypography: {
+      fontSize: '16px',
+      lineHeight: '19px',
+    },
+    content: {
+      padding: 0,
+    },
+    eyeNumber: {
+      color: theme.palette.text.secondary,
+      marginLeft: '6px',
+      fontSize: '9px',
+      lineHeight: '11px',
+    },
+    eyeIcon: {
+      width: '23px',
+      height: '13px',
+      marginLeft: '-3px',
+      color: '#6D6E71',
```

Comment:
pls use palette

Reply:
set a new property "secondary" in the palette.ts for this purpose